diff -up faad2-2.11.0/CMakeLists.txt.omv~ faad2-2.11.0/CMakeLists.txt
--- faad2-2.11.0/CMakeLists.txt.omv~	2023-11-07 13:45:34.515463753 +0100
+++ faad2-2.11.0/CMakeLists.txt	2023-11-07 13:45:50.497582164 +0100
@@ -5,6 +5,8 @@ cmake_minimum_required(VERSION 3.15)
 
 project(LIBFAAD2 LANGUAGES C)
 
+add_definitions(-DHAVE_LRINTF)
+
 # If FAAD is being bundled in another project, we don't want to
 # install anything. However, we want to let people override this, so
 # we'll use the FAAD_BUNDLED_MODE variable to let them do that; just
diff -up faad2-2.11.0/libfaad/common.h.omv~ faad2-2.11.0/libfaad/common.h
