diff -up faad2-2.8.8/common/mp4ff/Makefile.am.0001~ faad2-2.8.8/common/mp4ff/Makefile.am
--- faad2-2.8.8/common/mp4ff/Makefile.am.0001~	2019-01-08 17:24:43.462001840 +0100
+++ faad2-2.8.8/common/mp4ff/Makefile.am	2019-01-08 17:25:03.711319688 +0100
@@ -1,7 +1,7 @@
 noinst_LIBRARIES = libmp4ff.a
 noinst_HEADERS = mp4ff.h mp4ffint.h
 
-libmp4ff_a_CFLAGS = -DUSE_TAGGING=1
+libmp4ff_a_CFLAGS = -DUSE_TAGGING=1 -fPIC
 
 libmp4ff_a_SOURCES = mp4ff.c mp4atom.c mp4meta.c mp4sample.c mp4util.c \
 		     mp4tagupdate.c mp4ff.h mp4ffint.h
